book/Zapp-Pipes/protocols/login_and_payment_protocol.md

```diff
@@ -0,0 +1,46 @@
+# Login and Payment Protocol
+
+This document outlines the requirements for mapping the data to determine whether login and/or payment are required to gain access to a video item.
+
+## The "Free" flag
+Previously, the flag that was used to achieve this was the "free" flag in the extensions section of the item. However, there are a few drawbacks for using the "free" flag:
+1. The flag doesn't distinguish between whether an item requires login or payment to get access to, and might create some confusion. 
```

Comment:
```suggestion
1. The flag doesn't distinguish between whether access to an item requires login or payment, and this might create some confusion. 
```